### ISSUE TITLE
Visualize 3D Bounding Boxes Over Images and Lidar Data in ROS rviz

### ISSUE THREAD

**Issue by iamsiddhantsahu** [tianweiy/CenterPoint#18]
@tianweiy I was wondering how can we inference some of the pre-trained models provided in the MODEL ZOO on the test dataset of nuScenes and generate the co-ordinates of the **bounding box** for cars, pedestrians etc. And then how can we visualize them by overlaying them over **camera images** (front, front right, front left, back, back right and back left) and also in the **point cloud** visualization using **ros rviz**?

If would like to assist me with the above mentioned problem, I will be grateful!

**Comment by tianweiy** [comment]
```
inference some of the pre-trained models provided in the MODEL ZOO on the test dataset of nuScenes and generate the co-ordinates of the bounding box for cars, pedestrians etc.
```
For the test, you first need to generate the info file using [create_data.py](https://github.com/tianweiy/CenterPoint/blob/master/tools/create_data.py) and the version should be v1.0-test. After that, you do the inference in the standard way with [dist_test.py](https://github.com/tianweiy/CenterPoint/blob/master/tools/dist_test.py) by running 

```
python tools/dist_test.py CONFIG_NAME --work_dir SOME_PATH  --checkpoint SOME_PATH --speed_test --testset 
``` 
```
And then how can we visualize them by overlaying them over camera images (front, front right, front left, back, back right and back left) and also in the point cloud visualization using ros rviz
```

I never use ros rviz in the past. To draw on the camera images using matplotlib. you can modify the nuscenes-devkit's help function [Link](https://github.com/nutonomy/nuscenes-devkit/blob/274725ae1b3a2d921725016e3f4b383b8b218d3a/python-sdk/nuscenes/nuscenes.py#L903). 

I guess [This script](https://github.com/tianweiy/CenterPoint/blob/master/tools/multi_sweep_inference.py) will also be helpful if you want to use some ros libarary.

**Comment by iamsiddhantsahu** [comment]
@tianweiy Thanks! The same way `demo.py` uses the [**visual() function**](https://github.com/tianweiy/CenterPoint/blob/master/tools/demo_utils.py#L284) defined in `demo_utils.py`, can we not use this function to visualize? 
Additionally, I wanted to inquire can the `demo.py` file be used with **other configurations and pre-trained models**? And the [**detected points**](https://github.com/tianweiy/CenterPoint/blob/master/tools/demo.py#L97) in the demo.py file are **with respect to which co-ordinate system**?

**Comment by tianweiy** [comment]
You can use. But it will only visualize objects in Bird eye view. The devkit function also shows camera view. Yes, just change the path in demo.py and also in other configuration files change the data path accordingly. It is the reference frame coordinate (coordinate of the top lidar)

**Comment by iamsiddhantsahu** [comment]
This is addressed in much details in #19
